addons/mail/static/src/thread/message.xml

```diff
@@ -7,6 +7,9 @@
             <div t-if="!props.squashed" class="o-mail-avatar-container position-relative">
                 <img class="w-100 h-100 rounded-circle o_object_fit_cover o_redirect" t-att-class="{ 'cursor-pointer': props.hasActions }"
                     t-attf-src="/mail/channel/1/partner/{{message.authorId}}/avatar_128"/>
+                <div class="d-flex justify-content-center flex-shrink-0 position-absolute bottom-0 end-0 align-items-center rounded-circle bg-100 small o-muted">
+                    <PartnerImStatus partner="author"/> <!-- TODO openchat func associated with partner -->
```

Comment:
Remove comment.
It's preferable to have such comment in pad somewhere else, rather than in code.

Also this feature is not on the im status icon itself, but the avatar. It just happens in master we have different handlers for image and im status icon, but it maps to the same feature (opening chat of corresponding partner).
Ideally a click on avatar opens chat. The IM status icon should be part of the avatar.

But ok for the mean time to not introduce such component that combines im status icon and avatar.
Will do in follow-up PR.